File: Iniciante/URI 1008.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var funcionarios = parseInt(lines.shift());
var horas = parseInt(lines.shift());
var valorPorHora = parseFloat(lines.shift());
var salario = horas * valorPorHora;

console.log("NUMBER = " + funcionarios);
console.log("SALARY = U$ " + salario.toFixed(2));

File: Iniciante/URI 1008.r
input = file('stdin', 'r')

funcionarios = as.integer(readLines(input, n=1))
horas = as.integer(readLines(input, n=1))
valorPorHora = as.double(readLines(input, n=1))
salario = horas * valorPorHora

write(sprintf("NUMBER = %d", funcionarios), '')
write(sprintf("SALARY = U$ %.2f", salario), '')